config/default.yml

```diff
@@ -198,6 +198,12 @@ PreCommit:
     required_executable: 'grep'
     flags: ['-IHn', '^<<<<<<<\s']
 
+  Pep257:
```

Comment:
I'm comfortable with the idea of making this a hard failure, with users having the choice to opt-out. The PEP specification could use some help with adoption.